packages/sdk/src/cosmwasmclient.ts

```diff
@@ -150,13 +151,19 @@ export class CosmWasmClient {
    * @param address returns data for this address. When unset, the client's sender adddress is used.
    */
   public async getNonce(address?: string): Promise<GetNonceResult> {
-    const account = (await this.restClient.authAccounts(address || this.senderAddress)).result.value;
+    const account = await this.getAccount(address);
     return {
-      accountNumber: account.account_number,
-      sequence: account.sequence,
+      accountNumber: account ? account.account_number : 0,
+      sequence: account ? account.sequence : 0,
     };
   }
 
+  public async getAccount(address?: string): Promise<CosmosSdkAccount | undefined> {
```

Comment:
At some point I'd like to have custom return types for all the CosmWasmClient methods instead of returning the REST response directly (as it was done for `GetNonceResult`). But this is good for now. For blocks we don't have custom types as well.